src/environments/environment.prod.ts

```diff
@@ -0,0 +1,4 @@
+export const environment = {
+  production: true,
+  oauth2ClientId: 'nhlu5b6dvqkfvt9guc2nr0pdrn',
```

Comment:
Is it safe to commit oauth2ClientId?

Reply:
There is no point to hide it, because it's easy discoverable via DevTools in browser. Also it can't be used on other site because it has only one allowed redirect URL.